README.md

```diff
@@ -129,17 +129,55 @@ and run dump command:
 docker-compose run --rm db pg_dump -Fc -h db -v -U postgres feinstaub -f /opt/code/feinstaub-api-db.dump
 ```
 
---- 
+---
 
 ## Deployement
 
 To deploy the API, we use Dokku.
 
+### Postgres
+```
+sudo docker pull postgres:9.6
+
+sudo docker tag postgres:9.6 dokku/postgres:9.6
+
+dokku apps:create postgres
+
+sudo docker volume create --name feinstaub-db-data
+
+dokku docker-options:add postgres run,deploy --volume feinstaub-db-data:/var/lib/postgres
+
+dokku tags:deploy postgres 9.6
+
+# create database if non exists
+
+dokku enter postgres
+
+> createdb feinstaub -U postgres
 ```
 
+### API
+
+```
 dokku apps:create sensors-aq-api
 
-dokku proxy:ports-set sensors-aq-api http:80:8000
+dokku config:set sensors-aq-api \
+    API_SECRET_KEY=... \
+    API_FORECAST_IO_KEY=...
+
+#Link to postgres
+dokku docker-options:add sensors-aq-api run,deploy --link postgres.web.1:db
+
+#Bind mount for static
+dokku docker-options:add sensors-aq-api run,deploy --volume $(pwd)/../feinstaub-data:/home/uid1000
+
+dokku config:set sensors-aq-api HOST_DIR=$(pwd)/../feinstaub-data
+
+dokku proxy:ports-add sensors-aq-api http:8000:8000
+
+git remote add dokku dokku@api.aq.sensors.africa:sensors-aq-api
+
+git push dokku
```

Comment:
Let's separate these two commands out because it would happen on the local machine.